content-repo/extra-docs/articles/CI-CD-Article.md

```diff
@@ -99,9 +103,25 @@ For general information about the CI/CD process, see [CI/CD FAQs](#cicd-faqs).
      
      **NOTE:**  You can download directly from the development server. If downloading Automations and integrations, you need to use the `demisto-sdk split` command to split the content into the correct format.
 
+- **Migrate detached out-of-the-box content**
+  
+  If you have out-of-the-box  content from a content pack that is detached (such a Phishing incident type) and want to use the CI/CD process, run the following command:
+
+  `demisto-sdk download --system --type <object to download such as layouts, playbooks, scripts> -i <the name of the object to download> -o <the output directory. 
+  It is recommended to download to a new SystemPacks directory>`
```

Comment:
We need to elaborate on storing the detached items in the System folder - 
 - We are doing it to avoid validations errors.
 - We still need to store it in the SystemPacks folder as a Pack structure.
 - In the pack, we will store just the detached items and not all other files (like pack_metadata, release notes, readme, etc.)